backend/src/kitconcept/intranet/locales/de/LC_MESSAGES/kitconcept.intranet.po

```diff
@@ -39,23 +51,47 @@ msgstr "Blockkonfiguration"
 msgid "Can only be seen and edited by the owner."
 msgstr "Kann nur vom Eigentümer angesehen und bearbeitet werden."
 
-#: kitconcept/intranet/controlpanels/intranet.py:31
+#: kitconcept/intranet/controlpanels/intranet.py:33
 msgid "Custom CSS"
 msgstr "Benutzerdefiniertes CSS"
 
-#: kitconcept/intranet/controlpanels/intranet.py:32
+#: kitconcept/intranet/controlpanels/intranet.py:34
 msgid "Custom CSS for the intranet."
 msgstr "Benutzerdefiniertes CSS für das Intranet."
 
+#: kitconcept/intranet/controlpanels/intranet.py:50
+msgid "Default Feedback Email"
+msgstr "Standard-Feedback-E-Mail"
+
+#: kitconcept/intranet/controlpanels/intranet.py:44
+msgid "Disabled"
+msgstr "Behindert"
```

Comment:
@Tishasoumya-02 what is the context for this?

Reply:
It is a setting for person squared.

 
<img width="1046" height="288" alt="Screenshot 2025-09-01 at 4 02 32 PM" src="https://github.com/user-attachments/assets/cf66e8d1-c712-417c-ab23-3ed156b1df6b" />